src/typechecker/FStar.TypeChecker.Rel.fs

```diff
@@ -312,6 +324,17 @@ let singleton wl prob smt_ok     = {wl with attempting=[prob]; smt_ok = smt_ok}
 let wl_of_guard env g            = {empty_worklist env with attempting=List.map snd g}
 let defer reason prob wl         = {wl with wl_deferred=(wl.ctr, reason, prob)::wl.wl_deferred}
 let defer_lit reason prob wl     = defer (Thunk.mkv reason) prob wl
+let find_user_tac_for_uvar env (u:ctx_uvar) : option<sigelt> =
+    match u.ctx_uvar_meta with
+    | Some (Ctx_uvar_meta_attr a) ->
+      let hooks = Env.lookup_attr env Const.resolve_implicits_attr_string in
```

Comment:
Do we need `Const.resolve_implicit_attrs_string` or can we directly lookup the attribute `a` in the `env`?

Reply:
Particularly if we intend to allow multiple attributes on a binder, I want a way to interpret some attributes as being relevant for resolve implicits and others remaining for other purposes. So, I see the `resolve_implicits a` as a marker indicating that `a` should be interpreted by the resolve implicits mechanism; other attributes could be handled in other ways.